cmd/kubeseal/main.go

```diff
@@ -143,6 +149,13 @@ func readSecret(codec runtime.Decoder, r io.Reader) (*v1.Secret, error) {
 	return &ret, nil
 }
 
+func isMultiDocumentYAML(src []byte) bool {
```

Comment:
We could use this from `pkg/multidocyaml` if it wasn't private.

Reply:
I moved this function to that package so that it could be used by kubeseal and the contoller